scripts/rook.sh

```diff
@@ -37,6 +38,40 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/common.yaml"
 }
 
+function create_block_pool() {
+    curl -o newpool.yaml "${ROOK_URL}/pool-test.yaml"
+    sed -i "s/replicapool/$ROOK_BLOCK_POOL_NAME/g" newpool.yaml
+	kubectl create -f "./newpool.yaml"
+	rm -f "./newpool.yaml"
+
+    for ((retry = 0; retry <= ROOK_DEPLOY_TIMEOUT; retry = retry + 5)); do
+		echo "Checking RBD ($ROOK_BLOCK_POOL_NAME) stats... ${retry}s" && sleep 5
+
+		TOOLBOX_POD=$(kubectl -n rook-ceph get pods -l app=rook-ceph-tools -o jsonpath='{.items[0].metadata.name}')
+		TOOLBOX_POD_STATUS=$(kubectl -n rook-ceph get pod "$TOOLBOX_POD" -ojsonpath='{.status.phase}')
+		[[ "$TOOLBOX_POD_STATUS" != "Running" ]] && \
+			{ echo "Toolbox POD ($TOOLBOX_POD) status: [$TOOLBOX_POD_STATUS]"; continue; }
+
+		if kubectl exec -n rook-ceph "$TOOLBOX_POD" -it -- rbd pool stats "$ROOK_BLOCK_POOL_NAME" &>/dev/null; then
+			echo "RBD ($ROOK_BLOCK_POOL_NAME) is successfully created..."
+			break
+		fi
+	done
+
+	if [ "$retry" -gt "$ROOK_DEPLOY_TIMEOUT" ]; then
+		echo "[Timeout] Failed to get RBD pool $ROOK_BLOCK_POOL_NAME stats"
+		exit 1
+	fi
+	echo ""
+}
+
+function delete_block_pool() {
+	curl -o newpool.yaml "${ROOK_URL}/pool-test.yaml"
+    sed -i "s/replicapool/$ROOK_BLOCK_POOL_NAME/g" newpool.yaml
+	kubectl delete -f "./newpool.yaml"
+	rm -f "./newpool.yaml"
```

Comment:
do we need to check in toolbox that pool is deleted?

Reply:
This is provided for completeness, so not adding any checks as such (like create pool)